content/docs/1.7.0/deploy/uninstall/_index.md

```diff
@@ -142,18 +141,5 @@ you can cancel the uninstallation as the following.
 
 Redeploy the (same version) Longhorn App. Follow the uninstallation procedure above.
 
-#### Problems with CRDs
-
-If your CRD instances or the CRDs themselves can't be deleted for whatever reason, run the commands below to clean up. Caution: this will wipe all Longhorn state!
-
-```shell
-# Delete CRD finalizers, instances and definitions
-for crd in $(kubectl get crd -o jsonpath={.items[*].metadata.name} | tr ' ' '\n' | grep longhorn.rancher.io); do
-  kubectl -n ${NAMESPACE} get $crd -o yaml | sed "s/\- longhorn.rancher.io//g" | kubectl apply -f -
-  kubectl -n ${NAMESPACE} delete $crd --all
-  kubectl delete crd/$crd
-done
-```
-
```

Comment:
Shall we delete the commands? Could we replace `longhorn.rancher.io` with `longhorn.io`?

Reply:
I don't have enough technical context to be sure yet, but if the commands are no longer needed then deletion is good.

Reply:
I think we should keep this since it is related to a case where the CRs somehow can't be deleted.